src/api/teams/edit.rs

```diff
@@ -0,0 +1,59 @@
+use super::*;
+use crate::params;
+
+#[derive(serde::Serialize)]
+pub struct EditTeamBuilder<'octo, 'a> {
```

Comment:
Edit and Create seem extremely similar, with the only difference I can see is the send method. If that's the case I would recommend merging these two structs, and changing `slug` to be `Option<String>` and then use edit if present, and else use create. If the structs are merged I would also move this into `src/api/teams.rs` directly.

Reply:
There are 2 parameters in `CreateTeamBuilder` that don't exist in `EditTeamBuilder`: `maintainers` and `repo_names`. Is there a way to merge the structs while staying type-safe?

Reply:
Okay, I missed that there were two 2 parameters that were different,  it is fine then that they remain separate.